sdk/storage/storage-file-datalake/README.md

```diff
@@ -0,0 +1,520 @@
+    - From the overview page of your AAD Application, note down the `CLIENT ID` and `TENANT ID`. In the "Certificates & Secrets" tab, create a secret and note that down.
+    - Make sure you have AZURE_TENANT_ID, AZURE_CLIENT_ID, AZURE_CLIENT_SECRET as environment variables to successfully execute the sample(Can leverage process.env).
+
+  ```javascript
+  const { DefaultAzureCredential } = require("@azure/identity");
+  const { DataLakeServiceClient } = require("@azure/storage-file-datalake");
+
+  // Enter your storage account name
+  const account = "<account>";
+  const defaultAzureCredential = new DefaultAzureCredential();
+
+  const datalakeServiceClient = new DataLakeServiceClient(
+    `https://${account}.dfs.core.windows.net`,
+    defaultAzureCredential
+  );
+  ```
+
+  See the [Azure AD Auth sample](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-blob/samples/javascript/azureAdAuth.js) for a complete example using this method.
+
+  [Note - Above steps are only for Node.js]
+
+- Alternatively, you instantiate a `DataLakeServiceClient` with a `StorageSharedKeyCredential` by passing account-name and account-key as arguments. (account-name and account-key can be obtained from the azure portal)
+  [ONLY AVAILABLE IN NODE.JS RUNTIME]
+
+  ```javascript
+  const { DataLakeServiceClient, StorageSharedKeyCredential } = require("@azure/storage-file-datalake");
+
+  // Enter your storage account name and shared key
+  const account = "<account>";
+  const accountKey = "<accountkey>";
+
+  // Use StorageSharedKeyCredential with storage account and account key
+  // StorageSharedKeyCredential is only avaiable in Node.js runtime, not in browsers
+  const sharedKeyCredential = new StorageSharedKeyCredential(account, accountKey);
+  const datalakeServiceClient = new DataLakeServiceClient(
+    `https://${account}.dfs.core.windows.net`,
+    sharedKeyCredential
+  );
+  ```
+
+### Create a new file system
+
+Use `DataLakeServiceClient.getFileSystemClient()` to get a file system client instance then create a new file system resource.
+
+```javascript
+const { DefaultAzureCredential } = require("@azure/identity");
+const { DataLakeServiceClient } = require("@azure/storage-file-datalake");
+
+const account = "<account>";
+const defaultAzureCredential = new DefaultAzureCredential();
+
+const datalakeServiceClient = new DataLakeServiceClient(
+  `https://${account}.dfs.core.windows.net`,
+  defaultAzureCredential
+);
+
+async function main() {
+  // Create a file system
+  const fileSystemName = `newfilesystem${new Date().getTime()}`;
+  const fileSystemClient = datalakeServiceClient.getFileSystemClient(fileSystemName);
+  const createContainerResponse = await fileSystemClient.create();
+  console.log(`Create file system ${fileSystemName} successfully`, createContainerResponse.requestId);
+}
+
+main();
+```
+
+### List the containers
+
+Use `DataLakeServiceClient.listFileSystems()` function to iterate the containers,
```

Comment:
>containers [](start = 70, length = 10)

same as above, containers should be file systems?

Reply:
Yes, should be file system. Technically, they are the same. But in datalake SDK, we call it as file system. Updated.